backend/src/apiserver/client_manager.go

```diff
@@ -127,6 +134,31 @@ func (c *ClientManager) init() {
 
 	c.swfClient = client.CreateScheduledWorkflowClientOrFatal(
 		getStringConfig(podNamespace), getDurationConfig(initConnectionTimeout))
+
+	port, err := strconv.Atoi(getStringConfig(mysqlServicePort))
+	if err != nil {
+		glog.Fatalf("Failed to parse valid MySQL service port from %q: %v", getStringConfig(mysqlServicePort), err)
+	}
+
+	cfg := &mlpb.ConnectionConfig{
+		Config: &mlpb.ConnectionConfig_Mysql{
+			&mlpb.MySQLDatabaseConfig{
+				Host:     proto.String(getStringConfig(mysqlServiceHost)),
+				Port:     proto.Uint32(uint32(port)),
+				Database: proto.String("mlmetadata"),
+				User:     proto.String("root"),
+			},
+		},
+	}
+
+	mlmdStore, err := mlmetadata.NewStore(cfg)
+	if err != nil {
+		glog.Fatalf("Failed to create ML Metadata store: %v", err)
+	}
+	metadataStore := metadata.NewStore(mlmdStore)
+	runStore := storage.NewRunStore(db, c.time, metadataStore)
```

Comment:
Do we use a single DB for both the current tables and the new tables? 

Or are there two DBs?

Reply:
We use a separate db for isolation. They are not related to each other. Schema upgrades etc will anyway happen separately.